docs/reference/commandline/service_create.md

```diff
@@ -105,6 +106,21 @@ ID            NAME    REPLICAS  IMAGE        COMMAND
 4cdgfyky7ozw  redis   5/5       redis:3.0.7
 ```
 
+### Create a service with secrets
+Use the `--secret` flag to use a [secret](secret_create.md).  The following
```

Comment:
Use the `--secret` flag to give a container access to a secret. The following command....